src/main/java/com/mindera/school/spaceshiprent/persistence/entity/RentEntity.java

```diff
@@ -28,10 +29,12 @@ public class RentEntity {
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "user_id")
+    @ToString.Exclude
```

Comment:
do you really need this?

Reply:
well, i think i do. I think you added the @ToString annotation on this class and my intellij was complaining that it could cause memory consumption issues due to lazy loaded relationships and I trusted its judgement. 
can you explain this better to me?